docs/quickstart.rst

```diff
@@ -6,46 +6,71 @@ Quickstart
+Before starting this tutorial, first `install Docker on your machine <https://docs.docker.com/v17.09/engine/installation/>`_.
+Next `install Docker Compose on your machine <https://docs.docker.com/compose/install/>`_.
 
+Then, grab the Docker quickstart from our Git repository.
 
-Next, grab the Vagrant box from our Git repo and let Vagrant do the setup of the Inmanta server.
+.. code-block:: sh
+
+    git clone https://github.com/inmanta/quickstart-docker.git
+    cd quickstart-docker
+
+Now that we have the needed docker files, we will need to get the `Inmanta quickstart project <https://github.com/inmanta/quickstart/blob/docker/project.yml>`_
+itself and checkout the docker branch of the quickstart project:
 
 .. code-block:: sh
 
-    git clone https://github.com/inmanta/quickstart-vagrant.git
-    cd quickstart-vagrant
-    ./make_keys.sh
-    vagrant up
+    git clone https://github.com/inmanta/quickstart.git --branch docker quickstart-project
+
+The quickstart project can now be found under the newly created `quickstart-project` directory.
+It will be the basis for this quickstart.
+The ``quickstart-project`` directory will also be shared with the Inmanta server container
+(mounted to ``/home/inmanta/quickstart-project``).
+We will come back to the files in this repository 
+
+Finaly, have Docker Compose deploy the quickstart environment:
 
-Vagrant will set up the Inmanta server and two VMs to experiment on.
-When Vagrant is ready, you should be able to open the dashboard at http://127.0.0.1:8888.
+.. code-block:: sh
 
-.. warning::
+    docker-compose up
 
-    When using Vagrant in combination with VirtualBox, there is a known issue with SSH keys.
-    If this problem occurs to you, add the following lines to the Vagrantfile:
+Docker Compose will set up the Inmanta server, a postgres server and two VM-like containers to experiment on.
+When Docker Compose is done deploying and Inmanta server is running, you will be able to open the dashboard at http://127.0.0.1:8888.
+When you see the following output, the Inmanta server is ready to be used:
 
-    .. code-block:: sh
+.. code-block:: sh
 
-        config.ssh.insert_key = false
+    inmanta_quickstart_server | inmanta.protocol.rest    DEBUG   Start REST transport
+    inmanta_quickstart_server | inmanta                  INFO    Server startup complete
 
-    Notice that you are using a default key, this is insecure.
+.. note::
 
-To get a shell on the Inmanta server:
+    docker-compose will lock the current terminal and use it for output from all 4 containers.
+    You will need to open a new terminal to continue with this quickstart
+
+To get an interactive shell on the Inmanta server (this will be needed later):
 
 .. code-block:: sh
 
-    vagrant ssh server
+    docker exec -it "inmanta_quickstart_server" bash
 
+The rest of the quickstart guide assumes commands are executed from the root path of the quickstart-docker git repository, unless noted otherwise.
```

Comment:
I think this should be emphasized a bit more

Reply:
resolved